app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -22,23 +22,26 @@ import ScatterPlot from '~/components/visualization/ScatterPlot'
 import StudyViolinPlot from '~/components/visualization/StudyViolinPlot'
 import DotPlot from '~/components/visualization/DotPlot'
 import Heatmap from '~/components/visualization/Heatmap'
+import GeneListHeatmap from '~/components/visualization/GeneListHeatmap'
 import GenomeView from './GenomeView'
 import ImageTab from './ImageTab'
 import { getAnnotationValues, getDefaultSpatialGroupsForCluster } from '~/lib/cluster-utils'
 import RelatedGenesIdeogram from '~/components/visualization/RelatedGenesIdeogram'
 import InferCNVIdeogram from '~/components/visualization/InferCNVIdeogram'
 import useResizeEffect from '~/hooks/useResizeEffect'
+import LoadingSpinner from '~/lib/LoadingSpinner'
 import { log } from '~/lib/metrics-api'
 
 
 const tabList = [
   { key: 'loading', label: 'loading...' },
   { key: 'scatter', label: 'Scatter' },
-  { key: 'annotatedScatter', label: 'Annotated Scatter' },
+  { key: 'annotatedScatter', label: 'Annotated scatter' },
   { key: 'correlatedScatter', label: 'Correlation' },
   { key: 'distribution', label: 'Distribution' },
   { key: 'dotplot', label: 'Dot Plot' },
   { key: 'heatmap', label: 'Heatmap' },
+  { key: 'geneListHeatmap', label: 'Precomputed heatmap' },
```

Comment:
The only thing that I find confusing about this is that we give the user the option to customize the name of the dropdown menu, but the tab name does not inherit it.  I feel like it should, but not necessarily enough to warrant a change request.

Reply:
ah, great point.  Yeah, the tab names right now are relatively hardcoded, but that's worth keeping in mind as this feature gets more usage